mautic/mautic
# (#4774) Update Contact Owner as part of campaign action

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/4774 https://github.com/mautic/mautic/issues/897
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
As a Mautic user I want to update contact owner as part of a campaign so that I can assign contacts to different owner

#### Steps to test this PR:
1. Create new campaign, launch campaign builder and choose segment
2. Select Action
3. Choose "update contact owner" and select user.
4. Launch campaign (`app/console mautic:campaigns:update` + `app/console mautic:campaigns:trigger`
5. Users in segment should have new owner
